RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 -----
 * [internal] Converted part of Comment model to Swift. Should be no functional changes. [#16898]
 * [*] Enables Support for Global Style Colors with Full Site Editing Themes [#16823]
+* [***] New Block: Embed block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3727]
```

Comment:
Not sure if it's standard but I noticed that other entries related to changes in the editor include `Block editor: ` message at the beginning, what do you think about including it too?

Reply:
Done in https://github.com/wordpress-mobile/WordPress-iOS/pull/16911/commits/6c10ffea92c45529f6de51580e1efcca3bc8ba00. 🙇‍♀️